semgrep/semgrep/config_resolver.py

```diff
@@ -457,34 +471,18 @@ def resolve_config(config_str: str) -> Dict[str, YamlTree]:
     return config
 
 
-def generate_config() -> None:
-    import requests  # here for faster startup times
+def generate_config(fd: IO, lang: Optional[str], pattern: Optional[str]) -> None:
+    config = DEFAULT_CONFIG
```

Comment:
I took out the existing get from `TEMPLATE_YAML_URL`, let me know if this is a bad idea. It looks like we rarely (never?) update that URL, and I figured having one less call here wasn't a bad thing: https://semgrep.dev/docs/faq/#what-network-requests-are-made.